hack/validation/test_release.py

```diff
@@ -0,0 +1,181 @@
+import csv
+import pytest
+
+CSLB_PATH = 'tsv_output/cslb.tsv'
+HLES_DOG1_PATH = 'tsv_output/hles_dog_1.tsv'
+HLES_DOG2_PATH = 'tsv_output/hles_dog_2.tsv'
+
+
+@pytest.fixture(scope='session')
+def cslb_data():
+    return parse_file(CSLB_PATH)
+
+
+@pytest.fixture(scope='session')
+def hles_dog1_data():
+    return parse_file(HLES_DOG1_PATH)
+
+
+@pytest.fixture(scope='session')
+def hles_dog2_data():
+    return parse_file(HLES_DOG2_PATH)
+
+
+def parse_file(path):
+    rows = []
+    with open(path) as f:
+        reader = csv.DictReader(f, delimiter='\t')
+        for row in reader:
+            rows.append(row)
+
+    return rows
+
+
+def test_cslb_date(cslb_data):
+    for row in cslb_data:
+        assert '2021' not in row['cslb_date']
+        assert '2020' in row['cslb_date']
+
+
+def test_spay_or_neuter_age(hles_dog2_data):
+    for row in hles_dog2_data:
+        assert 'dd_spay_or_neuter_age' in row
+        assert row['dd_spay_or_neuter_age'] in {'', '5', '99', '2', '4', '7', '6', '8', '3', '1'}
```

Comment:
can we sort these values? it's not particularly obvious to me at a glance that this list includes the numbers 1 through 8.